File: 240416/최고의 13위치/best-place-of-13.py
N = int(input())
board = [list(map(int, input().split())) for _ in range(N)]

max_cnts = 0
for i in range(N):
    for j in range(N - 2):
        max_cnts = max(max_cnts, board[i][j] + board[i][j + 1] + board[i][j + 2])

print(max_cnts)

File: 260209/최고의 13위치/best-place-of-13.cpp
#include <bits/stdc++.h>
using namespace std;

int main() 
{
    int N;
    cin >> N;

    vector<vector<int>> board(N, vector<int>(N));
    for (int i = 0; i < N; i++)
        for (int j = 0; j < N; j++) cin >> board[i][j];

    // Please write your code here.
    int max_cnts = 0;
    for(int i = 0; i < N; i++)
    {
        for(int j = 0; j < N - 2; j++)
        {
            max_cnts = max(max_cnts, board[i][j] + board[i][j + 1] + board[i][j + 2]);
        }
    }

    cout << max_cnts;
    return 0;
}